mount/losetup_linux.go

```diff
@@ -91,6 +91,11 @@ func setupLoopDev(backingFile, loopDev string, param LoopParams) (*os.File, erro
 	if err != nil {
 		return nil, errors.Wrapf(err, "could not open loop device: %s", loopDev)
 	}
+	defer func() {
```

Comment:
the change looks good. would you mind to update the comment from L88-L89? thanks

```
// To make Autoclear (LO_FLAGS_AUTOCLEAR) work, this function intentionally
// doesn't close this file handle on defer.		// doesn't close this file handle on defer.
```

Reply:
@fuweid Thanks, updated.

Reply:
Sorry for unclear message. I think The L88-L89 comment should be removed. The caller should handle the file handler.

 cc @mxpv @kzys

Reply:
Got it. Updated.